packages/features/ee/platform/pages/settings/members.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import Shell from "@calcom/features/shell/Shell";
+import { UserListTable } from "@calcom/features/users/components/UserTable/UserListTable";
+import { MembershipRole } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc/react";
+import useMeQuery from "@calcom/trpc/react/hooks/useMeQuery";
+
+const PlatformMembersView = () => {
+  const { data: user } = useMeQuery();
+
+  const { data: currentOrg, isPending } = trpc.viewer.organizations.listCurrent.useQuery();
+
+  const isOrgAdminOrOwner =
+    currentOrg &&
+    (currentOrg.user.role === MembershipRole.OWNER || currentOrg.user.role === MembershipRole.ADMIN);
+
+  const canLoggedInUserSeeMembers =
+    (currentOrg?.isPrivate && isOrgAdminOrOwner) || isOrgAdminOrOwner || !currentOrg?.isPrivate;
+
+  return (
+    <Shell
+      heading="Member management"
+      title="Platform members"
+      hideHeadingOnMobile
+      withoutMain={false}
+      subtitle="Manage the admins and members in your platform team"
```

Comment:
We should include information that if invited member is MEMBER then he / she cant create oauth clients and is read only, but to create oauth clients teammates need to be invited as ADMIN. Otherwise people will invite as MEMBER and then members cant create oauth clients.